NEWS.rst

```diff
@@ -2,6 +2,13 @@
 
 .. towncrier release notes start
 
+
+towncrier 22.22.0 (2022-12-21)
```

Comment:
I believe there's a typo here - `22.22.0` instead of `22.12.0`.

Reply:
Thanks. Much appreciated